python/res/job_queue/workflow_runner.py

```diff
@@ -20,12 +20,20 @@ def __init__(self, workflow, ert=None, context=None):
 
         self.__context = context
         self.__workflow_result = None
+        self.__workflow_executor = ThreadPoolExecutor(max_workers=1)
```

Comment:
Double under scored prefix is reserved for Python. Please use single underscore instead 🙏

Reply:
Ok, shall I change the existing ones too?

Reply:
A new issue spawned for existing ones #861